std/traits.d

```diff
@@ -7664,9 +7664,13 @@ template getSymbolsByUDA(alias symbol, alias attribute) {
                   .format(names[0]));
     }
 
+    // filtering not compiled members such as not accessible members
+    enum noNotCompiledMembers(string name) = (__traits(compiles, __traits(getMember, symbol, name)));
```

Comment:
yes sorry I'm not very good with English, how about `excludeInaccessiblemembers` or `noInaccessiblemembers`?